eng/mgmt/Directory.Build.Mgmt.targets

```diff
@@ -0,0 +1,8 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- <Import Project="$(NetSdkBuildTargetsDir)\core\_Directory.Build.targets" Condition=" Exists('$(NetSdkBuildTargetsDir)\core\_Directory.Build.targets') "/>
+  <Import Project="$(LibraryToolsFolder)\bootstrapTools\bootstrap.targets" /> -->
+
+  <Target Name="initt">
```

Comment:
Should this be deleted?

Reply:
Yes, this will be deleted. There are quite few targets that being used for testing purpose and I will do a major cleanup of unused code soon.